test/e2e/containers_conf_test.go

```diff
@@ -675,4 +677,39 @@ var _ = Describe("Verify podman containers.conf usage", func() {
 		Expect(result).Should(Exit(0))
 		Expect(result.OutputToString()).To(ContainSubstring("Path to the OCI-compatible binary used to run containers. (default \"testruntime\")"))
 	})
+
+	It("Set kube_generate_type", func() {
```

Comment:
I am OK with adding a system test only.  I usually only add system tests since they're exercised in gating.

Reply:
Yeah the e2e test is unhappy for some reason, I will look into that later and get this in with the system test on in the interest of time.